src/github.com/getlantern/flashlight/logging/logging_test.go

```diff
@@ -112,3 +114,15 @@ func TestLoggly(t *testing.T) {
 		assert.Equal(t, 100, len(result["message"].(string)))
 	}
 }
+
+func TestIncludeInSample(t *testing.T) {
```

Comment:
See my more recent comment.  This "catch" actually broke things in practice when using real MAC addresses.  I went back to the old algorithm because we know that it actually does only sample a subset of clients, even if it's not exactly the right percentage.